learn/generation/rag-for-hybrid/rag-for-hybrid-search.ipynb

```diff
@@ -0,0 +1,2569 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    def weight_by_alpha(sparse_embedding: Dict[str, List[Any]], dense_embedding: List[float], alpha: float) -&gt; Tuple[Dict[str, List[Any]], List[float]]:
</pre><p>You can import this from pinecone-text but understand if you are are trying to be more instructive</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='101'/>

Reply:
<p>Yeah I was trying to be a bit verbose here :)</p>